### PR TITLE
Fix typo in readme

### PR DESCRIPTION
**Pull request by nidbCN**

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -45,4 +44,4 @@ Glitch平台注册地址 https://glitch.com/
 ```shell
 curl --upload-file ./.data/qrcode.png https://transfer.sh/qrcode.png
 ```
-你会得到一个链接，把链接复制到浏览器里打开，用你要登陆那个账户扫描下面的二维码并授权即可
\ No newline at end of file
+你会得到一个链接，把链接复制到浏览器里打开，用你要登陆那个账户扫描下面的二维码并授权即可
```
Comment (aimerneige) [inline README.MD:47]: You must be kidding. You didn't change anything

**Review by aimerneige — COMMENTED** [review]
You change what???

File: README.MD (hunk 1 above)
Comment (nidbCN) [reply README.MD:47]: No, I change some thing exactly **NOT** important.

Comment (mzdluo123) [reply README.MD:47]: ？

**Review by mzdluo123 — COMMENTED** [review]
？

**Pull request closed without merging**